app/controllers/microposts_controller.rb

```diff
@@ -1,4 +1,5 @@
-class MicropostsController < ApplicationController
+require 'api/api_controller'
```

Comment:
do you really need it?)

Reply:
it is not working without require, but I an create that controller inside MicropostsController file

Reply:
fixed